assets/layers/recycling/recycling.json

```diff
@@ -0,0 +1,532 @@
+{
+  "id": "recycling",
+  "name": {
+    "en": "Recycling",
+    "nl": "Recycling"
+  },
+  "description": {
+    "en": "A layer with recycling containers and centres",
+    "nl": "Een laag met recyclagingcontainers en -centrums"
+  },
+  "source": {
+    "osmTags": "amenity=recycling"
+  },
+  "minzoom": 12,
+  "title": {
+    "render": {
+      "en": "Recycling facility",
+      "nl": "Recyclingfaciliteit"
+    },
+    "mappings": [
+      {
+        "if": "recycling_type=centre",
+        "then": {
+          "en": "Recycling centre",
+          "nl": "Recyclingcentrum"
+        }
+      },
+      {
+        "if": "recycling_type=container",
+        "then": {
+          "en": "Recycling container",
+          "nl": "Recyclingcontainer"
+        }
+      }
+    ]
+  },
+  "mapRendering": [
+    {
+      "location": [
+        "point",
+        "centroid"
+      ],
+      "icon": "./assets/layers/recycling/recycling-14.svg",
```

Comment:
Perhaps use `"icon":"circle:white;./assets/layers/recycling/recycling-14.svg"` to add a white circle backfill. This prevents weird rendering when two points overlap: 
![image](https://user-images.githubusercontent.com/1466478/154480505-130b8032-f19c-4dfc-a27d-c4e22055ad4d.png)

And it prevents the background layer to 'bleed through': 
![image](https://user-images.githubusercontent.com/1466478/154480607-a7da1ab8-9371-4d22-bb7b-be0f6ab8edbe.png)